docs2/site/docs/guides/known-issues.md

```diff
@@ -346,3 +293,112 @@ Also see discussion in [#1310](https://github.com/graphql-dotnet/graphql-dotnet/
 Prior to GraphQL.NET version 5, enumeration values were case insensitive matches, which
 did not meet the GraphQL specification. This has been updated to match the spec; to revert to the prior
 behavior, please see [issue #3105](https://github.com/graphql-dotnet/graphql-dotnet/issues/3105#issuecomment-1109991628).
+
+### Multiple instances of same graph type error
+
+You may encounter errors for the following nature:
+
+> A different instance of the GraphType 'WidgetGraphType' with the name 'Widget' has already been registered within the
+> schema. Please use the same instance for all references within the schema, or use GraphQLTypeReference to reference a
+> type instantiated elsewhere.
+
+This is caused when the same graph type class has multiple distinct instances used within the schema. This is a change
+in GraphQL.NET v7, and exists as protection to ensure that the schema is initialized properly. Note that this restriction
+does not apply to scalars.
+
+Below are some samples of what does not work in GraphQL.NET v7, along with solutions:
+
+```csharp
+// sample 1: manually creating instances without providing the same instance for each use
+public class MyGraphType : ObjectGraphType
+{
+    public MyGraphType()
+    {
+        // creates an instance of WidgetGraphType
+        Field("field1", new WidgetGraphType());
+        // creates another instance of the same class
+        Field("field2", new WidgetGraphType());
+
+        // solution:
+        Field<WidgetGraphType>("field1");
+        Field<WidgetGraphType>("field2");
+    }
+}
+
+// sample 2: adding an instance to a union graph type
+public class MyUnionGraphType : UnionGraphType
+{
+    public MyUnionGraphType()
+    {
+        // creates an instance, which will be different than the one use elsewhere
+        AddPossibleType(new WidgetGraphType());
+
+        // solution:
+        Type<WidgetGraphType>();
+    }
+}
+
+// sample 3: pulling an instance from DI
+public class MyUnionGraphType : UnionGraphType
+{
+    public MyUnionGraphType(WidgetGraphType widgetType)
+    {
+        // since graph types are typically registered as transients, this reference to
+        // WidgetGraphType will be different than other instances throughout the schema
+        // and the following code does not work
```

Comment:
```suggestion
        // Since graph types are typically registered as transients, this reference to
        // WidgetGraphType will be different than other instances throughout the schema
        // and the following code does not work.
```